src/components/DisplayPhoto.js

```diff
@@ -0,0 +1,26 @@
+import "./../styles/DisplayPhoto.css";
+
+function DisplayPhotos(props) {
+  return (
+    <section>
+      {props.photos.length === 0 ? (
+        <h2>No photos found!</h2>
+      ) : (
+        <>
+          <h2>Photos</h2>
+          {/* {console.log("kkk", props.photos)} */}
```

Comment:
Not sure if this is still being used, but if not then the commented code can be removed